01-Introduction and Goals.md

```diff
@@ -4,70 +4,24 @@
 
 ## Quality Goals
 
-* Entity data which is included in a client wiki's article content is eventually consistent with updates on the corresponding repo. Data updates from the repo are available to the client within [10 minutes](https://gerrit.wikimedia.org/r/plugins/gitiles/operations/puppet/+/e3423c8627067a8de8352b9b7f3c59870d9b47c5/modules/icinga/manifests/monitor/wikidata.pp#10).
```

Comment:
I like that this turned into one quality goal each for client and repo. Much clearer that way!